recipes/newrelic/infrastructure/windows.yml

```diff
@@ -122,6 +135,23 @@ install:
           # Ensure agent has enough time to start
           powershell -command 'Start-Sleep -s 10; $output = Get-Service "newrelic-infra"; if ( -not ($output -like "*newrelic-infra*") ) { Write-Error "The infrastructure agent has not started after installing. Please try again later, or see our documentation for installing manually https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/install-configure/install-new-relic."; Exit 31; }'
 
+    assert_agent_status_ok:
+      cmds:
+        - |
+          powershell -command '
+            Start-Sleep -s 5;
```

Comment:
@jlegoff 
It looks like we already have a 10s sleep in the start_infra step, perhaps we can remove this extra sleep?